src/Viz.js

```diff
@@ -523,7 +523,6 @@ export default class Viz {
             order: ${this.order.toString()}
             symbol: ${this.symbol.toString()}
             symbolPlacement: ${this.symbolPlacement.toString()}
-            offset: ${this.offset.toString()}
             ${variables}`.replace(/ {4}/g, '');
```

Comment:
Besides removing offset, we should add the **transform** that was its replacement. Indeed I think we're also missing **resolution**. 

It would also be great if we could refactor a bit this, so we could use a proper array with these SUPPORTED_PROPERTIES (eg. from https://github.com/CartoDB/carto-vl/blob/0880f32303cce95220aaaf481d9dda7ea3035f94/src/Viz.js#L38)

Reply:
This is a nice catch! I changed it accordingly and added more meaningful tests.